cmd/lncli/walletrpc_active.go

```diff
@@ -287,10 +287,11 @@ func bumpFee(ctx *cli.Context) error {
 	}
 
 	resp, err := client.BumpFee(ctxc, &walletrpc.BumpFeeRequest{
-		Outpoint:   protoOutPoint,
-		TargetConf: uint32(ctx.Uint64("conf_target")),
-		Immediate:  immediate,
-		Budget:     ctx.Uint64("budget"),
+		Outpoint:    protoOutPoint,
+		TargetConf:  uint32(ctx.Uint64("conf_target")),
+		Immediate:   immediate,
+		Budget:      ctx.Uint64("budget"),
+		SatPerVbyte: ctx.Uint64("sat_per_vbyte"),
```

Comment:
Ah must've been lost in the wash with the original version when budget replaced everything else.

Reply:
yeah thanks @ziggie1984 for pointing this out!